posts/MarkoB/2025-05-11-hibernate-search-8-0-0-Beta1.adoc

```diff
@@ -0,0 +1,161 @@
+:awestruct-tags: [ "Hibernate Search", "Lucene", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+:hsearch-doc-url-prefix: https://docs.jboss.org/hibernate/search/8.0/reference/en-US/html_single/
+:horm-doc-url-prefix: https://docs.jboss.org/hibernate/orm/7.0/userguide/html_single/Hibernate_User_Guide.html
+:hsearch-getting-started-orm-url-prefix: https://docs.jboss.org/hibernate/search/8.0/getting-started/orm/en-US/html_single/
+:hsearch-getting-started-standalone-url-prefix: https://docs.jboss.org/hibernate/search/8.0/getting-started/standalone/en-US/html_single/
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-version-family: 8.0
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id: 33109
+---
+
+We just published Hibernate Search 8.0.0.Beta1,
+the first beta release of the next major version of Hibernate Search.
+
+This version includes the first implementation of the type-safe field references and Hibernate Search's static metamodel,
+compatibility with the latest versions of Elasticsearch 9.0 and OpenSearch 3.0,
+and other small adjustments and dependency upgrades.
+
+== What's new
+
+[NOTE]
+====
+Hibernate Search 8.0 is still in its early stages of development:
+some features still need to be completed or may change in a backwards-incompatible way.
+====
+
+=== Dependency upgrades
+
+[[orm-version]]
+Hibernate ORM (link:{hsearch-jira-url-prefix}/HSEARCH-5367[HSEARCH-5367])::
+Hibernate Search targets the Hibernate ORM 7.0 series, which implements Jakarta Persistence 3.2.0.
+In particular, it is currently based on Hibernate ORM 7.0.0.CR1.
+[[lucene-version]]
+Lucene (link:{hsearch-jira-url-prefix}/HSEARCH-5373[HSEARCH-5373])::
+The Lucene-next backend now uses Lucene 10.2.1.
+[[elasticsearch-version]]
+Elasticsearch (link:{hsearch-jira-url-prefix}/HSEARCH-5361[HSEARCH-5361]/(link:{hsearch-jira-url-prefix}/HSEARCH-5365[HSEARCH-5365]::
+The Elasticsearch backend works with Elasticsearch 9.0 and 8.18, as well as other already compatible versions.
+[[opensearch-version]]
+OpenSearch (link:{hsearch-jira-url-prefix}/HSEARCH-5363[HSEARCH-5363]::
+The Elasticsearch backend works with OpenSearch 3.0, as well as other already compatible versions.
+[[others-version]]
+Others::
+* link:{hsearch-jira-url-prefix}/HSEARCH-5374[HSEARCH-5374]: Upgrade to Elasticsearch **client** 9.0.1
+* link:{hsearch-jira-url-prefix}/HSEARCH-5370[HSEARCH-5370]: Upgrade to GSON 2.13.1.Final
+* link:{hsearch-jira-url-prefix}/HSEARCH-5368[HSEARCH-5368]: Upgrade to Jandex 3.3
+* link:{hsearch-jira-url-prefix}/HSEARCH-5371[HSEARCH-5371]: Upgrade to Jackson 2.19.0
+
+[[static-metamodel]]
+=== Static metamodel
+
+Hibernate Search's link:{hsearch-doc-url-prefix}#static-metamodel[static metamodel] is a set of generated classes that represents the structure of each entity's index,
+thereby allowing to reference index fields in a type-safe way to create search queries through the link:{hsearch-doc-url-prefix}#search-dsl[Search DSL].
```

Comment:
Here's my regular nitpicky reminder that "allow to" is grammatically incorrect; only "allow `<somebody>` to" is correct :grin: 

```suggestion
thereby allowing type-safe references to index fields when creating search queries through the link:{hsearch-doc-url-prefix}#search-dsl[Search DSL].
```